### DISCUSSION TITLE
An empty line is added when the user opens or reloads the the document .

### DISCUSSION THREAD

**Discussion by Isini-Dananjana** [BitPhinix/slate-yjs#384] [General]
When using YJS with Plate, it adds a new line to the top of a shared document whenever a new user joins the collaboration session. I think this happens due to the following code found in yTextToSlateElement.
```
 const children =
    // eslint-disable-next-line @typescript-eslint/no-use-before-define
    delta.length > 0 ? delta.map(deltaInsertToSlateNode) : [{ text: '' }];

  return { ...yText.getAttributes(), children };

```
When multiple users are in the same document, this empty line that is added also gets synced, resulting in many empty lines.

Any idea how to resolve the issue?

**Comment by austinbuckler** [comment]
I wonder if we could leverage suspense here? I imagine in most cases people would want some kind of loading indicator present instead of a blank document.

**Reply by BitPhinix** [reply]
Suspense probably would be the correct approach. Happy to accept a pr
